Source/VMware.vSphereDSC/DSCResources/Cluster/Cluster.schema.psm1

```diff
@@ -124,7 +124,8 @@ Configuration Cluster {
     }
 
     Push-NullablePropertiesToDscResourceBlock -ResourceBlockProperties $haClusterProperties -NullableProperties $nullableHAClusterProperties
-    New-DscResourceBlock -ResourceName 'HACluster' -Properties $haClusterProperties
+    $resBlock = New-DscResourceBlock -ResourceName 'HACluster' -Properties $haClusterProperties
+	$resBlock.Invoke($haClusterProperties)
```

Comment:
Fix the spacing here. This applies to all places where the **$resBlock.Invoke** is used.